__tests__/app.test.js

```diff
@@ -120,3 +129,47 @@ describe("GET/api/articles/article_id", () => {
       });
   });
 });
+
+describe("GET/api/articles", () => {
+  test("returns status:200 and returns all data from all endpoints", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const { article } = res.body;
+
+        article.forEach((article) => {
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
```

Comment:
This is good! You're testing each object in the array because of the forEach. There are two things I'd recommend with this test to sure it up a bit more:
- If the array is empty then the _forEach_ will skip and none of the assertions will run, so include a check for _length_ to make sure it isn't empty
- If you change the assertions to the `typeof` style that you have used previously, it will hit 2 birds with one stone. That would cover a test that the property exists, and also check that it's value is the right data type